server/views/components/instructions/template.njk

```diff
@@ -14,7 +14,11 @@
         <label class="govuk-label govuk-label--l" for="instructions">
           {{ params.html | safe if params.html}}
           <div id="more-detail-hint" class="govuk-hint">
-            {{ params.hintText | safe if params.hintText else "These notes are sent to the practitioner in the allocation email. They will also be saved as an SPO Oversight contact in NDelius, which you can review and update before saving. Do not include links in the notes." }}
+            {% if true == laoCase %}
+              {{ params.hintText | safe if params.hintText else "These notes are sent to the practitioner in the allocation email. They will also be saved as an SPO Oversight contact in NDelius, which you can review and update before saving. Do not include links in the notes." }}
```

Comment:
This logic seems to be the wrong way round? should it be 
**if false == laoCase** then do as before otherwise new text.